### PR TITLE
setS3TransferAccelerate option support for s3 only

### PR DESCRIPTION
**Pull request by prakashsvmx**
Amazon S3 Transfer Acceleration endPoint support

`setS3TransferAccelerate(endPoint)`

### CODE REVIEW

**Comment by prakashsvmx** [comment]
Currently i could not test the real acceleration on a bucket for an object. I will test it and mark it ready for review.

**Comment by prakashsvmx** [comment]
Verified with s3 it uploads with `<bucket>.s3-accelerate.amazonaws.com` as endpoint.

File: examples/set-accelerate-end-point.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (kannappanr) [inline examples/set-accelerate-end-point.js:1]: Please remove the empty lines at the top

File: examples/set-accelerate-end-point.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+/*
+ * MinIO Javascript Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
```
Comment (kannappanr) [inline examples/set-accelerate-end-point.js:6]: year should be 2021

**Review by kannappanr — APPROVED** [review]
LGTM

**Review by kanagarajkm — APPROVED** [review]

**Pull request merged**